perceval/backends/core/mediawiki.py

```diff
@@ -251,32 +260,44 @@ def __fetch_pre1_27(self, from_date=None):
         def fetch_incremental_changes(namespaces_contents):
             # Use recent changes API to get the pages from date
             npages = 0  # number of pages processed
+            pages_done = []  # pages already retrieved in reviews API
+
             rccontinue = ''
             hole_created = True  # To detect that incremental is not complete
             while rccontinue is not None:
                 raw_pages = self.client.get_recent_pages(namespaces_contents, rccontinue)
                 data_json = json.loads(raw_pages)
+
+                rccontinue = None
                 if 'query-continue' in data_json:
                     # < 1.27
                     rccontinue = data_json['query-continue']['recentchanges']['rccontinue']
                 elif 'continue' in data_json:
                     # >= 1.27
                     rccontinue = data_json['continue']['rccontinue']
-                else:
-                    rccontinue = None
+
```

Comment:
I would keep the `else` block here. I think is more clear and if we have to change something else, we won't break anything.